extensions/spotify/src/main/java/app/revanced/extension/spotify/misc/fix/LoginRequestListener.java

```diff
@@ -59,7 +65,23 @@ private static LoginResponse getLoginResponse(@NonNull LoginRequest loginRequest
 
         boolean isInitialLogin = !loginRequest.hasStoredCredential();
         if (isInitialLogin) {
-            Logger.printInfo(() -> "Received request for initial login");
+            Logger.printInfo(() -> "Received initial login request");
```

Comment:
Consistent with the else log.
```suggestion
            Logger.printInfo(() -> "Received request for initial login");
```